src/sentry/notifications/types.py

```diff
@@ -112,14 +129,36 @@ class NotificationSettingOptionValues(ValueEqualityEnum):
     COMMITTED_ONLY = 40
 
 
+class NotificationSettingsOptionEnum(Enum):
+    DEFAULT = "default"
+    NEVER = "never"
+    ALWAYS = "always"
+    SUBSCRIBE_ONLY = "subscribe_only"
+    COMMITTED_ONLY = "committed_only"
+
+
 NOTIFICATION_SETTING_OPTION_VALUES = {
-    NotificationSettingOptionValues.DEFAULT: "default",
-    NotificationSettingOptionValues.NEVER: "never",
-    NotificationSettingOptionValues.ALWAYS: "always",
-    NotificationSettingOptionValues.SUBSCRIBE_ONLY: "subscribe_only",
-    NotificationSettingOptionValues.COMMITTED_ONLY: "committed_only",
+    NotificationSettingOptionValues.DEFAULT: NotificationSettingsOptionEnum.DEFAULT.value,
+    NotificationSettingOptionValues.NEVER: NotificationSettingsOptionEnum.NEVER.value,
+    NotificationSettingOptionValues.ALWAYS: NotificationSettingsOptionEnum.ALWAYS.value,
+    NotificationSettingOptionValues.SUBSCRIBE_ONLY: NotificationSettingsOptionEnum.SUBSCRIBE_ONLY.value,
+    NotificationSettingOptionValues.COMMITTED_ONLY: NotificationSettingsOptionEnum.COMMITTED_ONLY.value,
 }
 
+NOTIFICATION_SETTING_V2_CHOICES = [
+    NotificationSettingsOptionEnum.ALWAYS.value,
+    NotificationSettingsOptionEnum.NEVER.value,
+    NotificationSettingsOptionEnum.SUBSCRIBE_ONLY.value,
+    NotificationSettingsOptionEnum.COMMITTED_ONLY.value,
+]
```

Comment:
```suggestion
NOTIFICATION_SETTING_V2_CHOICES = [ e.value for e in NotificationSettingsOptionEnum]
```

do we not need default here?

Reply:
@JoshFerge It seems for now we don't need a default. Default basically means you can delete the setting.